### PR TITLE
Add camel-soap wrapper

### PR DESCRIPTION
**Pull request by lopushen**

### CODE REVIEW

File: components/camel-soap/pom.xml [hunk 1]
```diff
@@ -37,6 +37,8 @@
             org.apache.camel*;version=${camel.version}
         </camel.osgi.export>
         <camel.osgi.import>
+            !org.w3._2003._05.soap_envelope,
+            !org.xmlsoap.schemas.soap.envelope,
```
Comment (lopushen) [inline components/camel-soap/pom.xml]: Added this exclusions because these packages are included in the component and are required

Comment (essobedo) [reply components/camel-soap/pom.xml]: This is not the right approach indeed if you look at the content of the artifact `camel-soap`, you will see that it contains those packages so the right approach is to export them

Comment (lopushen) [reply components/camel-soap/pom.xml]: understood, works this way, fixed, thanks!

**Review by essobedo — APPROVED** [review]
LGTM

**Pull request merged**